controllers/googleAuth.js

```diff
@@ -0,0 +1,10 @@
+function redirect(req, res) {
+  res.redirect("/");
+}
+
```

Comment:
We will be implementing a User dashboard so I think we should redirect it to that page only. What do you think?

Reply:
Yes, I think it will be good.